instances/UBERONParcellation/auditoryCortex.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/auditoryCortex",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "definition": "The auditory cortex consists of neocortical areas that are functionally activated predominantly to sound, with the shortest latency.",
+  "description": "The auditory cortex is the part of the temporal lobe that processes auditory information in humans and other vertebrates. It is a part of the auditory system, performing basic and higher functions in hearing, such as possible relations to language switching.It is located bilaterally, roughly at the upper sides of the temporal lobes – in humans on the superior temporal plane, within the lateral fissure and comprising parts of Heschl's gyrus and the superior temporal gyrus, including planum polare and planum temporale (roughly Brodmann areas 41, 42, and partially 22). Unilateral destruction, in a region of the auditory pathway above the cochlear nucleus, results in slight hearing loss, whereas bilateral destruction results in cortical deafness.",
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0100996",
+  "knowledgeSpaceLink": "https://knowledge-space.org/wiki/UBERON:0001393#auditory-cortex",
+  "name": "auditory cortex",
+  "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/UBERON_0001393",
+  "synonym": [
```

Comment:
I would kill synonyms completely since UBERON does not state any exact synonym.

Reply:
I disagree with that comment. We have added synonyms to our instances that UBERON did not state because they were simply missing there. "auditory area" is a synonym and very commonly used.